build/aliases.yaml

```diff
@@ -36,6 +36,13 @@ serve:
   - 'connect:livereload'
   - 'concurrent:serve'
 
+start:
```

Comment:
this is to compile for development purpose avoiding the test 🙏 . for me it is very convinient